crates/pathfinder/src/storage/state.rs

```diff
@@ -245,13 +245,15 @@ impl StarknetBlocksTable {
     /// Insert a new [StarknetBlock]. Fails if the block number is not unique.
     pub fn insert(connection: &Connection, block: &StarknetBlock) -> anyhow::Result<()> {
         connection.execute(
-            r"INSERT INTO starknet_blocks ( number,  hash,  root,  timestamp)
-                                   VALUES (:number, :hash, :root, :timestamp)",
+            r"INSERT INTO starknet_blocks ( number,  hash,  root,  timestamp, gas_price, sequencer_address)
```

Comment:
Minor not: double-spacing alignment
```suggestion
            r"INSERT INTO starknet_blocks ( number,  hash,  root,  timestamp,  gas_price,  sequencer_address)
```